src/OrbitGgp/include/OrbitGgp/Client.h

```diff
@@ -48,7 +49,7 @@ class Client {
 };
 
 [[nodiscard]] std::chrono::milliseconds GetClientDefaultTimeoutInMs();
-ErrorMessageOr<std::unique_ptr<Client>> CreateClient(
+ErrorMessageOr<std::pair<std::unique_ptr<Client>, Account>> CreateClient(
```

Comment:
Nit: I think this warrants the creation of a struct containing both the client and the account.

Reply:
Bot looking at this more closely, I'm not sure anymore if that's the best approach. There are a lot of variables called `client` which should probably be renamed because `client->first` or `client->client` is confusing. Same goes for this function.

Have you considered making the Account retrieval an Async function similar to the `GetWhateverAsync` functions?

Reply:
Wesome. LGTM!